### PR TITLE
fixes #3962

### LINKED ISSUE #3962: CMFPlone must not depend on plone.api
Dependency in setup.py, used in 

https://github.com/plone/Products.CMFPlone/blob/5f866f56d979ca0879181823a1a802035f727079/Products/CMFPlone/controlpanel/browser/error_log_form.py#L6

plone.api depends on CMFPlone, not vice versa!

### PR DESCRIPTION
**Pull request by jensens**
#3962 CMFPlone must not depend on plone.api

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@gforcada can we have these plone/meta tests for circular dependencies on CMFPlone too?

File: Products/CMFPlone/controlpanel/browser/error_log_form.py [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 from Products.CMFPlone.utils import safe_nativestring
 from Products.Five import BrowserView
 
-import plone.api as api
-
 
 class ErrorLogUpdate(BrowserView):
     def __call__(self):
-        member = api.user.get_current()
+        portal_membership = getToolByName("portal_membership")
```
Comment (davisagli) [inline Products/CMFPlone/controlpanel/browser/error_log_form.py:10]: @jensens Doesn't getToolByName need a context?

**Review by davisagli — CHANGES_REQUESTED** [review]

File: Products/CMFPlone/controlpanel/browser/error_log_form.py (hunk 1 above)
Comment (jensens) [reply Products/CMFPlone/controlpanel/browser/error_log_form.py:10]:
```suggestion
        portal_membership = getToolByName(self.context, "portal_membership")
```

Comment (jensens) [reply Products/CMFPlone/controlpanel/browser/error_log_form.py]: Indeed.

**Pull request merged**